src/jetstream/plugins/kubernetes/main.go

```diff
@@ -59,6 +59,19 @@ func (c *KubernetesSpecification) Register(echoContext echo.Context) error {
 	return c.portalProxy.RegisterEndpoint(echoContext, c.Info)
 }
 
+func (c *KubernetesSpecification) Validate(userGUID string, cnsiRecord interfaces.CNSIRecord, tokenRecord interfaces.TokenRecord) error {
+	response, err := c.portalProxy.DoProxySingleRequest(cnsiRecord.GUID, userGUID, "GET", "api/v1/pods")
```

Comment:
We might want to apply a `limit` param of `1` to this (see https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#list-all-namespaces-pod-v1-core), otherwise we could get a huge amount of data back